integration-cli/docker_cli_save_load_test.go

```
@@ -104,6 +104,10 @@ func (s *DockerSuite) TestSaveImageId(c *check.C) {
 	out, _ = dockerCmd(c, "images", "-q", repoName)
 	cleanedShortImageID := strings.TrimSpace(out)
 
+	// Make sure IDs are not empty
+	c.Assert(cleanedLongImageID, check.Not(check.Equals), "", check.Commentf("Id should not be empty."))
+	c.Assert(cleanedShortImageID, check.Not(check.Equals), "", check.Commentf("Id should not be empty."))
```

This is to fail early :sweat_smile: